flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -32,12 +35,14 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
-		9705A1C41CF9048500538489 /* Embed Frameworks */ = {
+		19C134552876922700263A48 /* Embed Frameworks */ = {
 			isa = PBXCopyFilesBuildPhase;
 			buildActionMask = 2147483647;
 			dstPath = "";
 			dstSubfolderSpec = 10;
 			files = (
+				19C134542876922700263A48 /* backend_bridge_fw.xcframework in Embed Frameworks */,
+				19C1345828769D8300263A48 /* tflite_backend_fw.xcframework in Embed Frameworks */,
```

Comment:
Xcode links all of the bundled libraries automatically.
I tried adding second instance of TFLite backend to the app. Xcode emitted a warning about duplicated symbols but there were no errors.
I think it shouldn't because we load libraries an its exported functions manually, and stuff inside libraries should already be linked to itself. But I'm not 100% sure.